src/config.ts

```diff
@@ -10,7 +10,7 @@ export const INIT_AMOUNT = Number(process.env.INIT_AMOUNT) || 0.01
 
 // Portal Setup
 export const PORTAL_API_URL =
-  process.env.PORTAL_API_URL || 'https://api.portalhq.io'
+  process.env.PORTAL_API_URL || 'https://api-staging.portalhq.io'
```

Comment:
any reason for updating this to be staging?

Reply:
I had just done that locally and accidentally pushed it, apologies.

Reply:
all good!